version/version.go

```diff
@@ -26,11 +26,11 @@ import (
 
 const (
 	// Version is the Operator's version
-	Version = "1.0.0"
+	Version = "1.1.0-dev"
 
 	// AstarteVersionConstraintString represents the range of supported Astarte versions for this Operator.
 	// If the Astarte version falls out of this range, reconciliation will be immediately aborted.
-	AstarteVersionConstraintString = ">= 0.10.0, < 1.1.0"
+	AstarteVersionConstraintString = ">= 0.10.0, < 1.2.0"
```

Comment:
Do we want to keep support for 0.10 here? I'm favourable, I'd say 1.2 should kill support for anything < 1.0 AND Voyager, this would also allow us to clean up a lot of crappy code we have in next iteration. @bettio ?

Reply:
agreed